bahmni-docker/docker-compose.yml

```diff
@@ -175,13 +175,59 @@ services:
       - 'openmrsdbdata:/var/lib/mysql'
     logging: *log-config
 
+  crater-php:
+    image: bahmni/crater-php:${CRATER_PHP_IMAGE_TAG:?}
+    profiles: ['default', 'crater']
+    restart: unless-stopped
+    working_dir: /var/www/
+    environment:
+      APP_URL: ${CRATER_APP_URL:?}
+      DB_HOST: ${CRATER_DB_HOST:?}
+      DB_PORT: ${CRATER_DB_PORT:?}
+      DB_DATABASE: ${CRATER_DB_DATABASE:?}
+      DB_USERNAME: ${CRATER_DB_USERNAME:?}
+      DB_PASSWORD: ${CRATER_DB_PASSWORD:?}
+      SANCTUM_STATEFUL_DOMAINS: ${CRATER_SANCTUM_STATEFUL_DOMAINS:?}
+      SESSION_DOMAIN: ${CRATER_SESSION_DOMAIN:?}
+      AUTO_INSTALL: ${CRATER_AUTO_INSTALL:?}
+      ADMIN_NAME: ${CRATER_ADMIN_NAME:?}
+      ADMIN_EMAIL: ${CRATER_ADMIN_EMAIL:?}
+      ADMIN_PASSWORD: ${CRATER_ADMIN_PASSWORD:?}
+      COMPANY_NAME: ${CRATER_COMPANY_NAME:?}
+      COMPANY_SLUG: ${CRATER_COMPANY_SLUG:?}
+      COUNTRY_ID: ${CRATER_COUNTRY_ID:?}
+
+  crater-nginx:
+    image: bahmni/crater-nginx:${CRATER_NGINX_IMAGE_TAG:?}
+    profiles: ['default', 'crater']
+    restart: unless-stopped
+    ports:
+      - 80:81
```

Comment:
This needs to be swapped `81:80`, syntax is <host_port>:<container_port>

Reply:
Okay